src/index.html

```diff
@@ -1,13 +1,338 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Nothing</title>
+    <link
+      rel="shortcut icon"
+      href="./images/icons/logo.png"
+      type="image/x-icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="on"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;700&family=Space+Mono&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__container">
+        <div class="header__content">
+          <div class="header__top top">
+            <a href="#" class="top__logo">
+              <img
+                class="top__image"
+                src="./images/icons/logo.png"
+                alt="logo"
+              >
+            </a>
+            <div class="top__icons">
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
```

Comment:
Need to fix the format phone everywhere only in href
```suggestion
              <a href="tel:+12345555555" class="icon icon--phone">
```